docs/source/fetching/persisted-queries.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Persisted Queries
+---
+
+Apollo supports features called "Persisted Queries" and "Automatic Persisted Queries" (APQs), to avoid sending the same large query documents over and over, and to allow safelisting of queries.
+
+Each query or mutation is identified by the SHA256 hash of its contents that can be sent to the server in lieu of sending a full query document. For APQs specifically, if the hash can't be found by the server, it sends back an error indicating that it needs the full query. If it receives this specific error, Apollo iOS will automatically retry the operation with the full query document without you having to do anything.
+
+### Support
+
+| | **Persisted Queries** | **Automatic Persisted Queries** |
+| - | :---------------: | :-------------------------: |
+| **GraphOS** | Yes | No |
```

Comment:
Not sure what it means to say Automatic Persisted Queries isn't supported in GraphOS? Would it be better to specify Gateway vs. Router vs. Server here?

Reply:
This is definitely one of the areas I'm looking for input for what the messaging should be here. We could definitely specify Gateway vs Server vs Router versions, which would give a more detail view of the support.

Reply:
Generally, I'm a fan of tables, but wondering if just listing out requirements might be clearer here.

I believe for PQ it's just:
- GraphOS (Enterprise plan)
- Router >= v1.25.0

For APQ, is it just:
- Apollo Server (not sure of version)

I might be misunderstanding why we would call it support though.